recipe/meta.yaml

```diff
@@ -158,12 +158,13 @@ outputs:
         - libabseil {{ libabseil }}       # [not win]
         # on osx, libuv supports torch.distributed support. See build.sh.
         - libuv 1.44.2                    # [win or osx]
-        - numpy {{ numpy }}
+        - numpy 2.0.0
```

Comment:
The latest we have available that also has py313 support is 2.1.3. Could we use that or do we prefer to use 2.0.0 for some reason?

Reply:
Should it be `numpy 2`?

Reply:
well I think we want to pin against exact versions in host in general for reproducibility.

As for 2.1 vs 2.0, it doesn't matter, because this version of pytorch doesn't support python 3.13 anyway, and both will cover the compatibility range we want, from something like 1.20 to 2.x for all x